geth/node/node.go

```diff
@@ -117,6 +117,7 @@ func defaultEmbeddedNodeConfig(config *params.NodeConfig) *node.Config {
 
 	if config.BootClusterConfig.Enabled {
 		nc.P2P.StaticNodes = makeBootstrapNodes(config.BootClusterConfig.BootNodes)
+		nc.P2P.BootstrapNodes = makeBootstrapNodes(config.BootClusterConfig.BootNodes)
```

Comment:
ideally "true" bootnodes should not be added as static nodes, it won't crash a p2p server but it will make server periodically try to connect with bootnode as a regular peer and it won't be able to do so

in prev pr i suggested to load static peers into BootClusterConfig.StaticNodes, but it is fine to merge it as is

Reply:
Yes, have seen, but they aren't so real static anymore, only in `p2p.Config` naming. We have two different meanings of *boot* in this context. 

I like your idea of separating it, have to find intelligent names.

Reply:
>  but they aren't so real static anymore, only in p2p.Config naming

maybe I misunderstood you but these nodes are called static even in p2p server code, not only config

Reply:
Yes, I only mentioned config because we're assigning it here. But from our perspective the formerly static compiled nodes now can be configured during startup. But yes, adopting the `geth` notation makes sense and I'll change it to `StaticNodes` and `BootstrapNodes`.

Reply:
i meant that they are called static not because they are compiled or configured with config, but because added statically without using discovery

Reply:
I suggest doing two additional things in this PR:
1. Change `BootClusterConfig` to `ClusterConfig` to avoid further confusion,
2. Define both `ClusterConfig.StaticNodes` and `ClusterConfig.BootNodes` and assign them accordingly to `P2P` object.

Reply:
Yes, I checked older code and problem starts with filling the `BootNodes` from different sources (the former static file and potential arguments, the latter overwrite the static file). And use it in different places (if enabled then for `BootstrapNodes` and always for `StaticNodes`, here earlier via `addPeer()`, now directly). So a clean separation definitely will help and also `StaticNodes` have to be set even when `BootClusterConfig.Enabled` is false.